pub_sub/csharp/order-processor/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+    app.UseDeveloperExceptionPage();
+}
+
+app.MapGet("/dapr/subscribe", () => {
+    var sub = new DaprSubscription(pubsub_name: "order_pub_sub", topic: "orders", route: "orders");
+    Console.WriteLine("Dapr pub/sub is subscribed to: " + sub);
+    return Results.Json(new DaprSubscription[]{sub});

```

Comment:
It would be better to use the declarative model for subscriptions. This is what most users are doing today and it has better scaling of concepts when your code is > 1 files.

https://github.com/dapr/dotnet-sdk/blob/master/examples/AspNetCore/RoutingSample/Startup.cs#L90

Reply:
Moved to [Dapr.Topic] attribute per our chat.  thank you!